src/review-form.ts

```diff
@@ -72,6 +81,10 @@ export class ReviewForm extends LitElement {
     if (changed.has('oldReview') && this.oldReview?.stars) {
       this.currentStars = this.oldReview.stars;
     }
+
+    if (changed.has('recaptchaManager') && this.recaptchaManager) {
+      this.setupRecaptcha();
```

Comment:
Something I just thought of... since setting up the recaptcha widget is asynchronous (ie has to load the library and such) and we can't submit the form until it's ready, we might want to have a `recaptchaReady` `@state` in here and disable the form until it's ready.